onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2602,18 +2595,21 @@ def aten_upsample_nearest1d_backward(
 @torch_op("aten::upsample_nearest2d", trace_only=True)
 def aten_upsample_nearest2d(
     self: TReal,
-    size: INT64,
+    output_size: INT64,
```

Comment:
Is this supposed to be optional?

Reply:
Not optional, as seen in the signature `"upsample_nearest2d(Tensor self, SymInt[2] output_size, float? scales_h=None, float? scales_w=None) -> Tensor`. I think the op was not very well defined which is why they are moving to use the `.vec` variant.